docs/how-to/create-slice.rst

```diff
@@ -0,0 +1,120 @@
+How to create a package slice for Chisel
+****************************************
+
+If your package doesn't yet have the slice definitions you actually need to **create your own slice definition** (which you can, later on, propose to be merged upstream for everyone else to use :ref:`publishslice` ).
+
+**Let's assume you are trying to create a slice definition for installing the OpenSSL binary into your ROCK!**
+
+Make sure the slice definition doesn't exist yet
+------------------------------------------------
+
+To avoid re-creating a slice that already exist, check the following to see if something that fits your needs already exists:
```

Comment:
```suggestion
To avoid re-creating a slice, check the following to see if something that fits your needs already exists:
```
This way we avoid repeating "exists"